### DISCUSSION TITLE
Deployment Guides from CI/CD with github actions as example

### DISCUSSION THREAD

**Discussion by aprilmintacpineda** [fastlane/fastlane#18295] [Show and tell]
I created the following:

- **iOS** https://github.com/aprilmintacpineda/react-native-guides/tree/master/fastlane_testflight
- **Android** https://github.com/aprilmintacpineda/react-native-guides/tree/master/fastlane_playstore

Which talks about using fastlane to deploy your React-Native app to PlayStore Internal track and Appstore TestFlight. Help me improve it by following the instructions for your new projects and validating the approach. Feel free to open a discussion there for any concerns or questions you have or problems you encounter while following the instructions.

**Comment by admcpr** [comment]
@aprilmintacpineda I get a 404 for both those links

**Reply by aprilmintacpineda** [reply]
done. Thanks.
